test/jdk/jdk/internal/math/FloatingDecimal/TestFloatingDecimal.java

```diff
@@ -119,11 +112,11 @@ private static int testAppendToDouble() {
             }
         }
 
-        return failures;
+        assertTrue(failures == 0);
```

Comment:
```suggestion
        assertEquals(0, failures);
```
Gives a better message because it explicitly knows both the expected value (1st arg) and the actual value (2nd arg).
Similarly for other `assertTrue` usages below).

Reply:
Good suggestion! Thanks. I'll change it.